cmd/server/server.go

```diff
@@ -43,12 +47,31 @@ func serverCmd(errorf errorFn) *cobra.Command {
 		Use:   "server",
 		Short: "Starts the mixer as a server",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Printf("Starting gRPC server on port %v", sa.port)
+			fmt.Printf("Starting gRPC server on port %v, type 'exit' to stop gracefully.\n", sa.port)
 
-			err := runServer(sa)
+			// The tracer will keep track of spans in memory and print them to stdout when the server finishes.
+			recorder := tracer.NewInMemoryRecorder()
```

Comment:
We definitely don't want to long term, but as a first impl for mixc/mixs I think it's fine for now (it's also the only way the basictracer impl records spans). Ultimately selecting the tracer will need to be config driven.

Another option would be implementing a simple recorder which dumps to stdout or glog and doesn't hold anything in memory. Personally I like this option and it simplifies some of the server code (no need to stop the server gracefully, so cmd/server/server.go goes back to almost exactly how it was before this PR).